ansible/roles/adm/tasks/main.yml

```diff
@@ -25,6 +25,13 @@
     state: present
   with_items: "{{ adm_logins | union(shell_logins) }}"
 
+- name: revoke users
+  user:
+    name: "{{ passwd[item].login }}"
+    state: absent
+    remove: yes
+  with_items: "{{ revoked_logins }}"
+
```

Comment:
FYI
- dropping users from passwd sometimes leads to issues in the future due to files belonging to unnamed UID
- "revocation" was already done as an [AllowUsers while-list](https://github.com/ooni/sysadmin/blob/master/ansible/roles/dom0-defaults/templates/sshd_config#L49-L50).

Reply:
Aah, thanks for pointing this out. I was not aware of the AllowUsers white-list. I will interrupt the execution of this and restart without the revoke step (I didn't reach that point of the playbook anyways).